README.md

```diff
@@ -46,6 +46,47 @@ Returns a JSON string representation of the given App Thinning Size Report.
 
 - `report_path` [String, required] Path to valid App Thinning Size Report text file.
 
+### `flag_android_violations`
```

Comment:
Since we are differentiating the Android version of the `flag_violations` method, let's also rename `flag_violations` to `flag_ios_violations`.